src/pages/PullRequestPage/subroute/FilesChangedTab/FilesChangedTab.jsx

```diff
@@ -59,7 +70,8 @@ function FilesChangedTab() {
       pullHeadCoverage: data?.pullHeadCoverage,
       pullBaseCoverage: data?.pullBaseCoverage,
       pullPatchCoverage: data?.pullPatchCoverage,
-    })
+    }) ||
+    isFirstPull
```

Comment:
Don't think this line could ever be reached since it's captured by the conditional in line 41

Reply:
ooops on it